planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
 
-## Site Title
+## Site Title (milestone)
 
 <!-- user story -->
 
-> "as a user I can read the title of the page"
+> "As a user I can read the title of the page"
 
 <!-- detailed description -->
 
-The site needs a clear title that's easy to read at the top of the page.
+<!--The site needs a clear title that's easy to read at the top of the page.-->
 
 <!-- acceptance criteria -->
 
 - [ ] _there is a prominent title at the top of the web page_
 
 <!-- code you think you will need -->
 
-### Site Title: HTML
+### Site Title HTML
 
-- Header with the web site's title
+- Header with the web site's title (issue)
 
-### Site Title: CSS
+### Site Title CSS
 
-- styling to center the title
+- styling to center the title(issue)
 
 ---
 
-## Introduction
+## Menu (milestone)
 
-> "As a user I can read an introduction to this project"
+> As a user I can read the details of the website from the Menu.
 
-It should be clear what this project is for and what someone can expect from the
-web page.
+- [ ] _A Menu contians Home,Introduction,Content and Extras are displayed at the
+      top of the page._
 
-- [ ] _there is a section directly under the navbar describing the project's
-      goals_
+### Menu HTML
+
+-[ ] Navbar:Home,Introduction,Content,Extras(issue)
+
+### Menu CSS
+
+- [ ] style elements, hover (issue)
+- [ ] place elements using flex (issue)
+
+---
+
+## Introduction (milestone)
+
+> "As a user I can read an introduction to this project" It should be clear what
+> this project is for and what someone can expect from the web page.
+
+- [ ] _there is a section "Introduction" in the navbar describing the project's
+      goal_
 
 ### Introduction: HTML
 
-- a section with some general information about the project and helpful links
+- a section with some general information about the project and helpful
+  links(issue)
 
 ### Introduction: CSS
 
-- styling to make the text look fancy
+- styling to make the text look fancy.(issue)
 
 ---
 
-## Main Info
+## Content (milestone)
 
 > "As a user I can learn about trees"
 
-There should be enough information that the user can learn the basics about
-trees.
+Since it is a home page we don't place much content but there should be links to
+the content with clear description.
+
+- [ ] _A section with some general information about the uses of trees,tree
+      health and social impact_(issue)
+- [ ] _there are links to different sources providing content about trees_
+
+### Content: HTML
+
+- a div with title and paragraph explaining why Batman is the best
+  superhero(issue)
+
+### Content: CSS
+
+- [ ] hover effect on links (issue)
+- [ ] styling to make the content look nice(issue)
+
+## Extra Info (milestone)
+
+> "As a user I can learn even more about trees" Some more links for students who
+> want to go beyond the basics.
+
+- [ ] _there is an aside with links for further reading_
+
+### Extra Info: HTML
+
+- An aside with links for digging deeper, some helpful text and a reference link
```

Comment:
We decided not to make aside part, Prince wanted to change it by photos.
Look at our design